arelle/plugin/validate/DBA/rules/fr.py

```diff
@@ -73,6 +73,80 @@ def rule_fr7(
     )
 
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_fr24(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    DBA.FR24: When cmn:TypeOfAuditorAssistance is "Revisionspåtegning" or "Auditor's report on audited financial statements"
+    then arr:DescriptionOfQualificationsOfAuditedFinancialStatements must not contain the following text:
+        - 'har ikke givet anledning til forbehold'
+        - 'has not given rise to reservations'
+    """
+    modelXbrl = val.modelXbrl
+    type_of_auditors_assistance_facts = modelXbrl.factsByQname.get(pluginData.typeOfAuditorAssistanceQn)
+    if type_of_auditors_assistance_facts is not None:
+        for auditor_fact in type_of_auditors_assistance_facts:
+            if (auditor_fact.xValid >= VALID and
+                    (auditor_fact.xValue == pluginData.auditedFinancialStatementsDanish or
+                     auditor_fact.xValue == pluginData.auditedFinancialStatementsEnglish)):
+                description_facts = modelXbrl.factsByQname.get(pluginData.descriptionOfQualificationsOfAuditedFinancialStatementsQn)
+                if description_facts is not None:
+                    for description_fact in description_facts:
+                        if description_fact.xValid >= VALID:
+                            for text in pluginData.hasNotGivenRiseToReservationsText:
+                                if text in str(description_fact.xValue):
+                                    yield Validation.error(
+                                        codes="DBA.FR24",
+                                        msg=_("The value of DescriptionOfQualificationsOfAuditedFinancialStatements must not "
+                                              "contain the text: \'{}\', when TypeOfAuditorAssistance is set to \'Revisionspåtegning\' "
+                                              "or \'Auditor's report on audited financial statements\'".format(text)),
```

Comment:
gettext translation needs to run on the static string:

```suggestion
                                              "or \'Auditor's report on audited financial statements\'").format(text),
```